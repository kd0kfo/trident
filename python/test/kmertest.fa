>KMerTest
CCTGGGTGCTGCAGGGTGCTAACTCTCTTGATGAGGCCGTCCTGGGTACTGTAGGGTACTGAGTCTCTGT
GATGAAGCCGTCCTGGGCTCTGTAGGGTGTTGAGTCTGAGTCTCTGTGATGAGGCCGTCCTGGGCCCTGT
AGGATGCTGAGTCTCTGTGATGAGGCCGTCCTGGACACTGTAGGGTGCTGAGTCTGAGTCTCTGTGATGA
GGCCGTCCTGGGCCCTGTAGGGTGTTAAGTCTCTGTGATGAGGCCGTCCTGGACACTGTAGGGTGCTGAG
TCTCTGTGATGAGGCCGTCCTGGGCACTGTAGGGTGTTGAGTCTGAGTCTCTGTGATGAGGCCGTCCTGG
GCCCTGTAGGGTGTTGAGTGTCTGTGATGAGGCCGTCCTGGGAGCTGTAGGGTGTTGACTCTGTGGTGGG
GCCGTCCTGGGCGCTGTAGGGTGCTGAGTCTCTGTGATGAGACCGTCCTGGGAGCTGTAGGGTGTTGACT
CTGTGGTGGGGCCATCCTGGGCGCTGTAGGGTGCTGAGTCTCTGTGATGAGACCGTCCTGGGAGCTGTAG
GGTGTTGACTCTGTGGTGGGGCCGTCCTGGGCGCTGTAGGGTGCTGAGTCTCTGTGATGAGGCCGTCCTG
GGAGCTGTAGGGTGTTGACTCTGTGGTGGGGCCGTCCTGGGCGCTGTAGGGTGTTGAGTCTCTGTGATGA
GGCCGTCCTGGGAGCTGTAGGGTGTTGACTCTGTGGTGGGGCCGTCCTGGGCGCTGTAGGGTGCTGAGGA
GCGTCCCTGGGCTCCACCCAGCAGATGCCAGTTCACCCCCATGTTGTGACATGCAGACATGTCCCCTGAT
ATTGCTGAGTGTGTCCGGGGAGGGGGCAGGATGTCCCCTGGCTGAGAACCACACTGCACTAAGGAATTAA
GAGGACGGCCGCACTTAGAGACACAGTTGCCCATGAGCTCATACCGGCTTCTCCAAGCAGGTCACTCTGA
AATCTTGTTTTTTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTGAGACAAAGTCGTGCTCTTGTCGCC
CAGGCTGGAGTGCGGTGGTGCCATCTCGGCTCACTGCAAGCTCCGCCTCCCGAGTTCACACCATTCTCCT
