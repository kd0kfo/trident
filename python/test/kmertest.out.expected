CTT	5
ATG	20
AAG	7
ACG	1
ATC	3
AAC	2
ATA	2
AGG	36
CCT	27
ACT	16
AGC	11
ACA	9
AGA	9
CAT	8
AAT	2
ATT	3
CTG	75
CTA	2
CTC	29
CAC	14
AAA	2
CCG	22
AGT	21
CAG	9
CAA	3
CCC	15
TAG	19
TAT	1
GGT	28
TGT	55
CGA	1
CCA	11
TCT	40
GAT	17
CGG	5
TTT	33
TGC	19
GGG	51
TGA	53
GGA	14
TGG	33
GGC	32
TAC	3
GAG	42
TCG	3
TTA	3
TTG	17
CGT	19
GAA	4
TAA	4
GCA	9
GTA	20
GCC	25
GTC	39
GCG	7
GTG	48
TTC	4
GTT	16
GCT	29
ACC	7
GAC	14
TCC	27
TCA	5
CGC	8
